README.md

```diff
@@ -6,42 +6,48 @@
 
 Coveo CLI is a command line interface to interact with the Coveo platform. It allows you to quickly create a [Coveo Headless](https://docs.coveo.com/headless)-powered search page for your Coveo organization, relying on [Angular](https://angular.io), [React](https://reactjs.org/) or [Vue.js](https://vuejs.org/).
 
-## Usage
+## Installation
```

Comment:
Putting "NPM/NPX" at the same level as "Mac", "Windows", and "Linux" is slightly confusing IMO, because it makes it seem like npm/npx is another OS.

I'd present the information as follows:

## Installation

### Option 1: Install via an Executable

Download and run the executable for your operating system to install the latest available version of the CLI.

Afterward, you can run `coveo update` at any time to update your CLI installation to the latest version.

- Mac
  ...

- Windows
  ...

- Linux
  ...

### Option 2: Install via npm

If you have [installed node-gyp](https://github.com/nodejs/node-gyp#installation), you can alternatively install the CLI globally via [npm](https://www.npmjs.com/package/@coveo/cli):

```sh
npm install -g @coveo/cli
```

Afterward, you can run `npm update -g @coveo/cli` at any time to update your npm-based CLI installation to the latest version.